.github/workflows/end-to-end.yml

```diff
@@ -48,4 +48,15 @@ jobs:
         api-key: ${{ secrets.MAESTRO_CLOUD_API_KEY }}
         app-file: /Users/runner/DerivedData/Build/Products/Debug-iphonesimulator/DuckDuckGo.app
         workspace: .maestro/privacy_tests
+
+    - name: Create Asana task when workflow failed
+        if: ${{ failure() }}
+        uses: honeycombio/gha-create-asana-task@main
+        with:
+          asana-secret: ${{ secrets.ASANA_ACCESS_TOKEN }}
+          asana-workspace-id: ${{ secrets.GH_ASANA_WORKSPACE_ID }} 
+          asana-project-id: ${{ secrets.GH_ASANA_IOS_APP_PROJECT_ID }} 
+          asana-section-id: ${{ secrets.GH_ASANA_INCOMING_ID }} 
```

Comment:
Does it have to be in Secrets? I would like to avoid polluting it with "safe" stuff.

Reply:
We can use GitHub Variables for that. An example is on our OpenSSL-XCFramework repo.

Here are definitions:
https://github.com/duckduckgo/OpenSSL-XCFramework/settings/variables/actions

And here is the usage example:
https://github.com/duckduckgo/OpenSSL-XCFramework/blob/main/.github/workflows/release.yml#L24

Reply:
AFAIK the diff between variables and secrets is that you can’t read back the latter from the UI. Anyway, this doesn’t look a bad thing to have for these things, so I’m moving to use them.